packages/smooth_app/lib/data_models/user_preferences.dart

```diff
@@ -91,4 +92,9 @@ class UserPreferences extends ChangeNotifier {
 
   bool? getFlag(final String key) =>
       _sharedPreferences.getBool(_getFlagTag(key));
+
+  set devMode(final int value) =>
+      _sharedPreferences.setInt(_TAG_DEV_MODE, value);
+
+  int get devMode => _sharedPreferences.getInt(_TAG_DEV_MODE) ?? 0;
```

Comment:
Also why not a bool

Reply:
Of course a `bool` would do the job in a first approach, but in my experience I noticed that sometimes we're stuck with a `bool` when we want to upgrade the app to something more elaborate. An `int` is safer.